test/FunctionalTests/Tests/DataOData/Tests/OData.Common.Tests/PublicApi/PublicApi.bsl

```diff
@@ -6148,6 +6163,12 @@ public sealed class Microsoft.OData.UriParser.WildcardSelectItem : Microsoft.ODa
 	public virtual T TranslateWith (SelectItemTranslator`1 translator)
 }
 
+public enum Microsoft.OData.UriParser.Aggregation.AggregateExpressionType : int {
+	EntitySetAggregate = 2
+	None = 0
+	PropertyAggregate = 1
+}
+
 public enum Microsoft.OData.UriParser.Aggregation.AggregationMethod : int {
 	Average = 3
 	CountDistinct = 4

```

Comment:
Nothing. GitHub just skipped some lines because there was no changes in it.
If you look at [here](https://github.com/athoscouto/odata.net/blob/entity_set_aggregation/test/FunctionalTests/Tests/DataOData/Tests/OData.Common.Tests/PublicApi/PublicApi.bsl#L6175), you'll see that 2 and 0 are unchanged.